### DISCUSSION TITLE
Institutional email verification

### DISCUSSION THREAD

**Discussion by malcom23** [RosettaCommons/rosetta#101] [ROSIE] [answered]
Hello, 
we have a small group of bionformatics working at the department of pediatry. W e would like to access ROSIE. 
our institution is https://www.burlo.trieste.it/ and emails end on ...@burlo.trieste.it
thanks

**Comment by lyskov** [comment, marked as answer]
Thank you for letting us know, - this should be fixed now. Please feel free to re-open if you still having issues.
